src/components/TeamMember.js

```diff
@@ -0,0 +1,46 @@
+import React, { useEffect } from 'react';
+import AOS from 'aos';
+import 'aos/dist/aos.css';
+import '../TeamMember.css';
+
+export default function TeamMember(props) {
+  useEffect(() => {
+    AOS.init({
+      duration: 2000,
+    });
+  });
+  const isRight = props.isRight;
+  if (isRight)
+    return (
+      <>
+        <div className=" h-64 w-full bg-gray-900 flex overflow-hidden my-10">
+          <div
+            className="member-img w-3/5 h-full bg-black "
+            data-aos="fade-right"
+          >
+            <img className="bg-cover bg-black" alt="" src=""></img>
```

Comment:
add alt and image should be recieved from props
```suggestion
            <img className="bg-cover bg-black" alt="profile" src={image}></img>
```